code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -555,8 +590,8 @@
 			src.animate_tail_stop()
 
 		if("beep")
-			if (!isipc(src))
-				src << span("notice", "You're not a machine!")
+			if (!check_has_mouth(src))

```

Comment:
This one should be !isipc(src) the vomit check was the one that needed check_has_mouth(src)

Reply:
this is why you don't code while piss-tired ree